pom.xml

```
@@ -59,20 +59,47 @@
         </plugins>
     </build>
     <dependencies>
-    <dependency>
-        <groupId>com.microsoft.azure</groupId>
-        <artifactId>azure-mgmt-keyvault</artifactId>
-        <version>1.11.1</version>
-    </dependency>
-    <dependency>
-        <groupId>com.microsoft.azure</groupId>
-        <artifactId>azure</artifactId>
-        <version>1.11.1</version>
-    </dependency>
+        <!-- https://mvnrepository.com/artifact/com.azure.resourcemanager/azure-resourcemanager -->
         <dependency>
-            <groupId>com.microsoft.azure</groupId>
-            <artifactId>azure-keyvault</artifactId>
-            <version>1.1</version>
+            <groupId>com.azure.resourcemanager</groupId>
+            <artifactId>azure-resourcemanager</artifactId>
+            <version>2.6.0</version>
+        </dependency>
+        <!-- https://mvnrepository.com/artifact/com.azure/azure-core -->
+        <dependency>
+            <groupId>com.azure</groupId>
+            <artifactId>azure-core</artifactId>
+            <version>1.18.0</version>
```

There's a new version of this library.
```suggestion
            <version>1.21.0</version>
```